packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js

```diff
@@ -260,12 +284,15 @@ export default class WebRTCMediaEngine {
         // Since gum is protected by @oneflight, returning it here will block
         // until it completes but, more importantly, propagate a thrown
         // exception up the stack
-        resolve(this._getUserMedia());
+        resolve(this._getUserMedia()
+          .then(tap(() => this.logger.info('gum completed'))));
         return;
       }
 
       resolve();
     })
+      .then(() => {
```

Comment:
Is this to get just another tick in?

Reply:
nope. looks like i pulled out some dev code, but missed the then block. will fix